src/objects/setup_configuration/tests/test_site_config.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+
+from django.contrib.sites.models import Site
+from django.db.models import QuerySet
+from django.test import TestCase
+
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+from django_setup_configuration.test_utils import execute_single_step
+
+from objects.setup_configuration.steps.sites import SitesConfigurationStep
+
+TEST_FILES = (Path(__file__).parent / "files").resolve()
+
+
+class SitesConfigurationStepTests(TestCase):
+    def test_empty_database(self):
+        test_file_path = str(TEST_FILES / "sites_empty_database.yaml")
+
+        execute_single_step(SitesConfigurationStep, yaml_source=test_file_path)
+
+        sites: QuerySet[Site] = Site.objects.all()
+
+        self.assertEqual(sites.count(), 2)
+
+        example_site: Site = sites.get(name="Example site")
+        self.assertEqual(example_site.domain, "example.com")
+
+        alternative_site: Site = sites.get(name="Alternative example site")
+        self.assertEqual(alternative_site.domain, "alternative.example.com")
+
+    def test_existing_sites(self):
+        test_file_path = str(TEST_FILES / "sites_existing_sites.yaml")
```

Comment:
Can't you reuse the same file `sites_empty_database.yaml` and rename it to `sites.yaml` or something instead?